paramak/parametric_components/tokamak_plasma_from_points.py

```diff
@@ -0,0 +1,73 @@
+import math
+from pathlib import Path
+
+import numpy as np
+import scipy
+
+from paramak import Plasma
+
+
+class Plasma(Plasma):
+    """Creates a double null tokamak plasma shape that is controlled
+       by 4 shaping parameters.
```

Comment:
Isn't it 3 parameters here ? Namely inner, outer and high point ? or does the hight point counted as 2 ?

Reply:
I was thinking the high and low point are the same, but I guess this is not always true

Reply:
They are symetrical indeed (in the case of an up-down symmetrical plasma). How do you wish to take vertical displacement into account ?